lib/iris/fileformats/netcdf.py

```diff
@@ -84,7 +84,7 @@
              'coordinates', '_FillValue', 'formula_terms', 'grid_mapping',
              'leap_month', 'leap_year', 'long_name', 'missing_value',
              'month_lengths', 'scale_factor', 'standard_error_multiplier',
-             'standard_name', 'units', 'valid_max', 'valid_min', 'valid_range']
```

Comment:
Your proposing that these attributes should **not** be treated as standard attributes on the cube/coord attributes dictionary?
I think that's what your are eluding to?
Being treated as special (part of this list) is what currently means the attribute is both not interpreted and is simply thrown away on load.

It may be that the at some point the cube undergoes an API change and does something special with valid_range etc. but I would propose that we provide real user benefit without interpreting these attributes just yet and simply not throwing them away.